NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -256,3 +415,517 @@ def make_sim_shower(corsika, observer=None, detector=None, station_id=None):
+            plt.scatter(self.obs_positions_vxB_vxvxB[:,0], self.obs_positions_vxB_vxvxB[:,1], c=max_efield, cmap='viridis', marker='o', edgecolors='k')
+            cbar = plt.colorbar()
+            cbar.set_label('max amplitude')
+            plt.xlabel('v x B [m]')
+            plt.ylabel('v x v x B [m]')
+            plt.show()
+            plt.close()
+
+        return self.fluence_interpolator
+
+    def plot_footprint_fluence(self, radius=300):
+        """
+        plots the interpolated values of the fluence in the shower plane
+
+        Parameters
+        ----------
+        radius : float
+            radius around shower core which should be plotted
+
+        save_file_path : str
+            if provided, figure will be stored there
+        """
+        
+
+        # Make color plot of f(x, y), using a meshgrid
+        ti = np.linspace(-radius, radius, 500)
+        XI, YI = np.meshgrid(ti, ti)
+
+        ### Get interpolated values at each grid point, calling the instance of interp2d_fourier
+        ZI =  self.fluence_interpolator(XI, YI)
+
+        # And plot it
+        maxp = np.max(ZI)
+        fig, ax = plt.subplots(figsize=(8, 6))
+        ax.pcolor(XI, YI, ZI, vmax=maxp, vmin=0, cmap=cm.gnuplot2_r)
+        mm = cm.ScalarMappable(cmap=cm.gnuplot2_r)
+        mm.set_array([0.0, maxp])
+        cbar = plt.colorbar(mm, ax=ax)
+        cbar.set_label(r'energy fluence [eV/m^2]', fontsize=14)
+        ax.set_xlabel(r'$\vec{v} \times \vec{B} [m]', fontsize=16)
+        ax.set_ylabel(r'$\vec{v} \times (\vec{v} \times \vec{B})$ [m]', fontsize=16)
+        ax.set_xlim(-radius, radius)
+        ax.set_ylim(-radius, radius)
+        ax.set_aspect('equal')
+        return fig, ax
+
+    def get_interp_efield_value(self, position_on_ground, core):
+        """
+        Accesses the interpolated electric field given the position of the detector on ground. For the interpolation, the pulse will be
+        projected in the shower plane. If the geomagnetic angle is smaller than 15deg, the electric field of the closest observer position is returned.
+
+        Parameters
+        ----------
+        position_on_ground : np.array (3)
+            position of the antenna on ground
+
+        core : np.array (3)
+            position of the core on ground
+
+        Returns
+        -------
+        efield_interp : float
+            interpolated efield value or efield of clostest observer
+
+        trace_start_time : float
+            start time of the trace
+        """
+        logger.debug(f"get interpolated efield for antenna position {position_on_ground} on ground and core position {core}")
+        antenna_position = copy.copy(position_on_ground)
+        #core and antenna need to be in the same z plane
+        antenna_position[2] = core[2]
+        # transform antenna position into shower plane with respect to core position, core position is set to 0,0 in shower plane
```

Comment:
What do you mean with "core position is set to 0,0 in shower plane" ?

Reply:
that we move our detector around, such that the relation of detector position and core is still the same, although the core in the shower plane is at x=0, y=0.